cloud-tasks/function/index.js

```diff
@@ -0,0 +1,102 @@
+/**
+ * Copyright 2019 Google LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+/**
+ * Responds to an HTTP request from Cloud Tasks and sends an email using data
+ * from the request body.
+ */
+exports.sendPostcard = async (req, res) => {
+  const sgMail = require('@sendgrid/mail');
+
+  // Get the SendGrid API key from the environment variable.
+  const key = process.env.SENDGRID_API_KEY;
+  if (!key) {
+    const error = new Error(
+      'SendGrid API key not provided as environment variable.'
+    );
+    error.code = 401;
+    throw error;
+  }
+  sgMail.setApiKey(key);
+
+  // Get the body from the Cloud Task request.
+  const postcardBody = req.body;
+  if (!postcardBody.to_email) {
```

Comment:
Instead of all these if statements, can we send the response like such?

```js
if (!postcardBody.fromEmail) return res.status(400).send('fromEmail not provided.');
...
```

This section should be ~4 lines long.